Osiris/Hacks/Misc.cpp

```diff
@@ -1080,6 +1081,50 @@ void Misc::voteRevealer(GameEvent& event) noexcept
     memory->clientMode->getHudChat()->printf(0, " \x0C\u2022Osiris\u2022 %c%s\x01 voted %c%s\x01", isLocal ? '\x01' : color, isLocal ? "You" : entity->getPlayerName().c_str(), color, votedYes ? "Yes" : "No");
 }
 
+void Misc::onVoteStart(const void*& data, int& size) noexcept
+{
+    if (!miscConfig.revealVotes)
+        return;
+
+    constexpr auto voteName = [](int index) {
+        switch (index)
+        {
+        case 0: return "Kick";
+        case 1: return "Change Level";
+        case 6: return "Surrender";
+        case 13: return "Start TimeOut";
+        default: return "ERROR";
+        }
+    };
+
+    ProtobufReader msg{ static_cast<const std::uint8_t*>(data), size };
+    const auto ent_idx = msg.readInt32(2);
+    const auto vote_type = msg.readInt32(3);
+
+    if (ent_idx && vote_type) {
```

Comment:
This check eliminates `vote_type` being `0` so `voteName` lambda will never return "Kick".